### ISSUE TITLE
JSDocs comments no longer higlighted

### ISSUE THREAD

**Issue by wolfy1339** [atom/language-javascript#600]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

With Tree-Sitter parser on, i no longer get any syntax highlighting for JSDoc comments

### Steps to Reproduce

1. Turn on tree-sitter
2. Open file with JSDoc comments

**Expected behavior:** Have syntax highlighting
**Actual behavior:** No syntax highlighting for JSDoc comments

**Reproduces how often:** 100%

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.
```
$ atom --version
Atom    : 1.33.0-nightly3
Electron: 2.0.10
Chrome  : 61.0.3163.100
Node    : 8.9.3
```
```
$ apm --version
internal/child_process.js:323
    throw errnoException(err, 'spawn');
    ^

Error: spawn UNKNOWN
    at _errnoException (util.js:1024:11)
    at ChildProcess.spawn (internal/child_process.js:323:11)
    at exports.spawn (child_process.js:502:9)
    at EventEmitter.<anonymous> (C:\Users\Thomas2\AppData\Local\atom\app-1.33.0-nightly3\resources\app\apm\lib\apm-cli.js:169:17)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)

```
### Additional Information

```
Community Packages (34)
├── atom-beautify@0.33.2 (disabled)
├── atom-discord@1.6.1
├── atom-easy-jsdoc@4.12.5
├── atom-html-preview@0.2.5 (disabled)
├── atom-ide-ui@0.13.0
├── atom-material-syntax@1.0.8
├── atom-material-syntax-dark@1.0.0
├── atom-material-ui@2.1.3
├── atom-rst-preview-docutils@1.0.0 (disabled)
├── atom-ternjs@0.18.3 (disabled)
├── code-peek@1.4.21
├── dockerletion@0.2.0 (disabled)
├── editorconfig@2.2.2
├── file-icons@2.1.25
├── ide-css@0.3.0
├── ide-json@0.2.1
├── ide-python@1.0.0 (disabled)
├── ide-typescript@0.7.6
├── ide-yaml@0.4.4 (disabled)
├── language-apache@1.8.0 (disabled)
├── language-batchfile@0.7.5 (disabled)
├── language-diff@0.7.0 (disabled)
├── language-docker@1.1.8 (disabled)
├── language-ignore@1.7.4
├── language-lua@0.9.11 (disabled)
├── language-restructuredtext@1.1.2 (disabled)
├── linter@2.2.0 (disabled)
├── linter-eslint@8.4.1
├── preview@0.17.0 (disabled)
├── react@0.18.0
├── react-es7-snippets@0.3.13 (disabled)
├── svg-preview@0.12.1 (disabled)
├── teletype@0.13.3 (disabled)
└── wakatime@7.0.5
```

**Comment by nebulousdog** [comment]
I can confirm this on `0.129.15`.

```
atom --version
Atom    : 1.32.1
Electron: 2.0.9
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

```
apm --version
apm  2.1.2
npm  6.2.0
node 8.9.3 x64
atom 1.32.1
python 2.7.15+
git 2.19.1
```

```
apm list
Built-in Atom Packages (92)
├── atom-dark-syntax@0.29.1
├── atom-dark-ui@0.53.3
├── atom-light-syntax@0.29.1
├── atom-light-ui@0.46.3
├── base16-tomorrow-dark-theme@1.6.0
├── base16-tomorrow-light-theme@1.6.0
├── one-dark-ui@1.12.5
├── one-light-ui@1.12.5
├── one-dark-syntax@1.8.4
├── one-light-syntax@1.8.4
├── solarized-dark-syntax@1.2.0
├── solarized-light-syntax@1.2.0
├── about@1.9.1
├── archive-view@0.65.1
├── autocomplete-atom-api@0.10.7
├── autocomplete-css@0.17.5
├── autocomplete-html@0.8.7
├── autocomplete-plus@2.41.0
├── autocomplete-snippets@1.12.0
├── autoflow@0.29.4
├── autosave@0.24.6
├── background-tips@0.28.0
├── bookmarks@0.45.1
├── bracket-matcher@0.89.3
├── command-palette@0.43.5
├── dalek@0.2.2
├── deprecation-cop@0.56.9
├── dev-live-reload@0.48.1
├── encoding-selector@0.23.9
├── exception-reporting@0.43.1
├── find-and-replace@0.215.14
├── fuzzy-finder@1.8.2
├── github@0.20.2
├── git-diff@1.3.9
├── go-to-line@0.33.0
├── grammar-selector@0.50.1
├── image-view@0.63.1
├── incompatible-packages@0.27.3
├── keybinding-resolver@0.38.4
├── line-ending-selector@0.7.7
├── link@0.31.6
├── markdown-preview@0.159.25
├── metrics@1.6.2
├── notifications@0.70.5
├── open-on-github@1.3.1
├── package-generator@1.3.0
├── settings-view@0.256.1
├── snippets@1.3.5
├── spell-check@0.74.1
├── status-bar@1.8.15
├── styleguide@0.49.12
├── symbols-view@0.118.2
├── tabs@0.109.2
├── timecop@0.36.2
├── tree-view@0.224.2
├── update-package-dependencies@0.13.1
├── welcome@0.36.7
├── whitespace@0.37.7
├── wrap-guide@0.40.3
├── language-c@0.60.11
├── language-clojure@0.22.7
├── language-coffee-script@0.49.3
├── language-csharp@1.1.0
├── language-css@0.43.0
├── language-gfm@0.90.5
├── language-git@0.19.1
├── language-go@0.46.4
├── language-html@0.51.5
├── language-hyperlink@0.16.3
├── language-java@0.30.0
├── language-javascript@0.129.15
├── language-json@0.19.2
├── language-less@0.34.2
├── language-make@0.22.3
├── language-mustache@0.14.5
├── language-objective-c@0.15.1
├── language-perl@0.38.1
├── language-php@0.44.0
├── language-property-list@0.9.1
├── language-python@0.51.8
├── language-ruby@0.72.12
├── language-ruby-on-rails@0.25.3
├── language-sass@0.62.0
├── language-shellscript@0.27.9
├── language-source@0.9.0
├── language-sql@0.25.10
├── language-text@0.7.4
├── language-todo@0.29.4
├── language-toml@0.18.2
├── language-typescript@0.4.9
├── language-xml@0.35.2
└── language-yaml@0.32.0

Community Packages (5) /home/tom/.atom/packages
├── busy-signal@1.4.3
├── intentions@1.1.5
├── linter@2.2.0
├── linter-eslint@8.4.1
└── linter-ui-default@1.7.1
```

**Comment by dancastellon** [comment]
I'm also not seeing JSDOC highlighting working.

**Comment by dancastellon** [comment]
A workaround: enable the `react` package -> settings -> check "Enabled for All Javascript Files"

**Comment by Arcanemagus** [comment]
@dancastellon As this is a Tree-sitter issue the correct workaround is to simply disable the Settings -> Core -> Use Tree Sitter Parsers option. 😉

**Comment by nebulousdog** [comment]
Hi @Arcanemagus, I can confirm that unchecking this works for me.

![image](https://user-images.githubusercontent.com/2218331/48100641-f2f30b00-e1d8-11e8-8def-9ff37bbc9e9f.png)

**Comment by maxbrunsfeld** [comment]
Hi everyone, just letting you know that the fix for this is in progress at https://github.com/atom/language-javascript/pull/629. I believe all that PR needs is some testing. Once it is done, we will probably publish the fix in a hotfix release, Atom 1.32.3 (and 1.33.0-beta3).

Sorry for the disruption.

**Comment by maxbrunsfeld** [comment]
Ok, JSDocs highlighting has been merged. I'm not sure if we'll do another hotfix release for this. If not, the next monthly release of Atom, 1.33.0, is supposed to ship at the end of November.
